sort/dutchsationalslagsort.go

```diff
@@ -0,0 +1,25 @@
+// Implementation of Dutch National Flag Sort algorithm
+// Reference: https://en.wikipedia.org/wiki/Dutch_national_flag_problem
+
+package sort
+
+import "github.com/TheAlgorithms/Go/constraints"
+
+// DutchNationalFlagSort sorts an array of integers with the Dutch National Flag algorithm
+func DutchNationalFlagSort[T constraints.Signed](arr []T) []T {
```

Comment:
Sort is in the name of the package - calling it `sort.DutchNationalFlag` suffices.

Reply:
fixed, thank you